include.js

```diff
@@ -5868,6 +5868,12 @@ let cards = [
     imageLink: './Art/animationzone/ezgif.com-gif-maker',
     author: 'Erez',
     githubLink: 'https://github.com/ErezAvni9'
+  },
+  {
```

Comment:
Hi @graphitexhd , please add the _**artName**_ property and give your art a great name!